.github/workflows/ci.yml

```diff
@@ -24,6 +24,8 @@ jobs:
   test-docs-build-verify:
     name: Test Verify Docs Build
     needs: pre-commit
+    # inheriting secrets does not work for forked repos
+    if: ${{ github.event.pull_request.head.repo.owner == 'beeware' }}
```

Comment:
Is this an intentional inclusion? It doesn't seem related to the caching problem...

Reply:
Apologies; I should have explicitly mentioned this.

It is intentional; the [first run](https://github.com/beeware/.github/actions/runs/4756507186) of CI failed because the `RTD_API_TOKEN` secret did not resolve. I considered removing the test altogether but this seemed like a better middle ground to ensure CI passes for PRs from forked repos going forward while the test can test at least run sometimes.

P.S. - when I originally added this test, I moved the PR from my fork to this repo; at the time, I was hoping that future PRs from forks wouldn't hit the same problem....I was wrong...